src/test/java/no/unit/alma/bibs/AlmaBibsIntegrationTest.java

```diff
@@ -23,7 +23,7 @@
     public static final String NOT_FOUND_ALMA_ERROR_CODE = "402203";
     public static final String NOT_FOUND_ALMA_ERROR_MESSAGE = "Input parameters mmsId %s is not valid.";
 
-    @Test
+    //@Test
```

Comment:
Skal denne testen fjernes?

Reply:
Tja, vet ikke. Jeg synes det er en god model for å bruke det ut i prosjketer. men den feiler jo uten secretId.